### ISSUE TITLE
2.0 bower.json Polymer version

### ISSUE THREAD

**Issue by H3dz** [RoXuS/paper-datatable-api#37]
It should be "polymer": "Polymer/polymer#^2.0.0-rc.1"

**Comment by nidelson** [comment]
There is already an entry to apply release 1:
```
  "resolutions": {
    "polymer": "^2.0.0-rc.1",
  }
```

What we can do is upgrade to release 3:
```
  "resolutions": {
    "polymer": "^2.0.0-rc.3",
  }
```
... and test.

**Comment by nidelson** [comment]
But I agree with this entry ...
```
"dependencies": {
    "polymer": "Polymer/polymer#^2.0.0-rc.3",
  },
```
... the component would stay up to date.
What do you think @RoXuS ?

**Comment by H3dz** [comment]
The other elements all have the dependency for the Release Candidate.

Currently running bower update will give the following notice:

Please note that,
    paper-datatable-api#d6011f0d90 depends on polymer#2.0-preview which resolved to polymer#d46694e5d5

Resort to using polymer#^2.0.0-rc.1 which resolved to polymer#2.0.0-rc.3
Code incompatibilities may occur.

**Comment by RoXuS** [comment]
Hello @H3dz, @nidelson,

I think that we have to be up to date so:
```
"dependencies": {
    "polymer": "Polymer/polymer#^2.0.0-rc.3",
  },
```

Should be ok ?

**Comment by nidelson** [comment]
That's it!

**Comment by RoXuS** [comment]
@H3dz in fact the polymer version is resolved to `d46694e5d5` because of `polymerfire`.

Polymerfire is for dev so we can't resolved to RC3.
